### PR TITLE
Bump deps

### PR DESCRIPTION
**Pull request by crudo**
Bump Lodash because of https://www.npmjs.com/advisories/577

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,10 +24,10 @@
   },
   "homepage": "https://github.com/stylelint/stylelint-checkstyle-formatter#readme",
   "devDependencies": {
-    "tape": "^4.2.2",
+    "tape": "4.9.1",
     "xml2js": "^0.4.15"
   },
   "dependencies": {
-    "lodash": "4.17.4"
+    "lodash": "4.17.10"
```
Comment (MhMadHamster) [inline package.json]: can we not specify the exact version for dependencies and devDependencies?

**Review by MhMadHamster — CHANGES_REQUESTED** [review]

**Comment by crudo** [comment]
I personally consider using tilde or caret as a bad practise, but as you wish :)

**Review by MhMadHamster — APPROVED** [review]

**Comment by crudo** [comment]
@MhMadHamster Would you please release new version?

**Comment by MhMadHamster** [comment]
done

**Pull request merged**